app/controllers/application_controller.rb

```diff
@@ -129,8 +134,15 @@ def skip_session_expiration
     @skip_session_expiration = true
   end
 
+  def valid_i18n_param
+    active_param = params[:locale]
+    return nil unless active_param.present? && I18n.available_locales.include?(active_param.to_sym)
+    active_param
+  end
```

Comment:
What do you think about moving this to a separate i18n param validator class? That's what Reek was suggesting, which I agree with. I'd prefer not to disable Reek offenses if there is an easy way to resolve them.